lib/node_modules/@stdlib/math/base/special/tan/lib/main.js

```diff
@@ -37,22 +37,18 @@
 var getHighWord = require( '@stdlib/number/float64/base/get-high-word' );
 var kernelTan = require( '@stdlib/math/base/special/kernel-tan' );
 var rempio2 = require( '@stdlib/math/base/special/rempio2' );
+var ABS_MASK = require( '@stdlib/constants/float64/high-word-abs-mask' );
+var EXPONENT_MASK = require( '@stdlib/constants/float64/high-word-exponent-mask' );
```

Comment:
Can we please keep the original variable names here, i.e. `HIGH_WORD_ABS_MASK` and `HIGH_WORD_EXPONENT_MASK`? That's more descriptive and matches more closely what we have in C, as well.

Reply:
Sure, I have changed that now. The reason I earlier used only `ABS_MASK` and `EXPONENT_MASK` was that we had the same in our `sin` implementation : https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/sin/lib/main.js#L37.

I'll be making a quick PR to change this in `sin` too, then.
Thanks!